src/java.base/share/classes/sun/security/jca/Providers.java

```diff
@@ -100,6 +100,20 @@ private Providers() {
         "SunJCE",
     };
 
+    // Hardcoded fully qualified names of providers to use for JAR
```

Comment:
This should be hyphenated: "fully-qualified class names".

Reply:
Changed.